packages/node_modules/@webex/plugin-meetings/src/statsAnalyzer/mqaUtil.js

```diff
@@ -8,11 +8,11 @@ export const getAudioReceiverMqa = ({audioReceiver, statsResults, lastMqaDataSen
   // Hop by hop are numbers and not percentage so we compare on what we sent the last min
   audioReceiver.common.mediaHopByHopLost = (statsResults[mediaType][sendrecvType].totalPacketsLost - lastMqaDataSent[mediaType][sendrecvType].totalPacketsLost) || 0;
   audioReceiver.common.rtpHopByHopLost = statsResults[mediaType][sendrecvType].totalPacketsLost - lastMqaDataSent[mediaType][sendrecvType].totalPacketsLost || 0;
-  audioReceiver.streams[0].common.rtpJitter = statsResults[mediaType][sendrecvType].rtpJitter;
+  audioReceiver.streams[0].common.rtpJitter = statsResults[mediaType][sendrecvType].jitter * 1000;
```

Comment:
Can we constantize the 1000 and 100 values with a descriptive name to indicate a bit better what's going on? why are we multiplying jitter for example, i imagine its because MQA wants it some other unit